docs/api-reference.md

```diff
@@ -148,9 +148,9 @@ Valid [resolver](types-reference.md#resolver) that is used to resolve the GraphQ
 InterfaceType.set_type_resolver(type_resolver)
 ```
 
-Sets type [resolver](types-reference.md#resolver) used to resolve the `str` with name of GraphQL type to which `obj` belongs to.
+Sets the type [resolver](types-reference.md#resolver) used to resolve the `str` with the name of the GraphQL type to which `obj` belongs to.
 
-Returns value passed to `type_resolver` argument.
+It returns value passed to `type_resolver` argument.
```

Comment:
Spelling change is unnecessary IMO, there's plenty of docs out there that put return value that way.